Uni_life_tracker/app/src/main/java/com/generic/ult/recyclerAdapter.java

```diff
@@ -0,0 +1,50 @@
+package com.generic.ult;
+
+import Entities.Event;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+
+public class recyclerAdapter extends RecyclerView.Adapter<recyclerAdapter.MyViewHolder>{
```

Comment:
The same of the class should be RecyclerAdapter, make sure to refactor when renaming